api/yarn.lock

```diff
@@ -1565,6 +1564,11 @@ color@3.0.x:
     color-convert "^1.9.1"
     color-string "^1.5.2"
 
+colorette@^1.2.1:
```

Comment:
co to jest ?

Reply:
> co to jest ?

nie mam pojecia, ale chyba dodalo sie po mergu z mainem